src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -1,76 +1,61 @@
-import { shallow, mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import React from "react";
 
-import SearchBox from "./SearchBox";
+import SearchBox, { Label } from "./SearchBox";
 
 describe("SearchBox ", () => {
   it("renders", () => {
-    const wrapper = shallow(<SearchBox />);
-    expect(wrapper).toMatchSnapshot();
+    render(<SearchBox />);
+    // Get the wrapping element for the snapshot.
+    // eslint-disable-next-line testing-library/no-node-access
+    expect(document.querySelector(".p-search-box")).toMatchSnapshot();
```

Comment:
Would getting by the "searchbox" role not work here?

Reply:
The searchbox role is only available on the text input but `.p-search-box` gets the wrapping element (which contains the label, input, buttons etc.)